app/main.py

```diff
@@ -44,6 +45,22 @@
     openapi_tags=tags_metadata,
 )
 
+app.add_middleware(
+    CORSMiddleware,
+    # FastAPI doc related to allow_origin (to avoid CORS issues):
+    # "It's also possible to declare the list as "*" (a "wildcard") to say that all are allowed.
+    # But that will only allow certain types of communication, excluding everything that involves
+    # credentials: Cookies, Authorization headers like those used with Bearer Tokens, etc.
+    # So, for everything to work correctly, it's better to specify explicitly the allowed origins."
+    # => Workaround: use allow_origin_regex
+    # Source: https://github.com/tiangolo/fastapi/issues/133#issuecomment-646985050
+    allow_origin_regex="https?://.*",
+    allow_credentials=True,
```

Comment:
Do not allow credentials, we have no need of it yet, and if we need it at a certain point, we would want to restrict CORS in this case.

```suggestion
    # This will exclude credentials (cookies, authorization headers, etc.) 
    # which is fine for us
    # If in the future you want to use allow-credentials, use `allow_origin_regex`
    # see: https://github.com/tiangolo/fastapi/issues/133#issuecomment-646985050
    allow_origin=["*"],
    allow_credentials=False,
```

Reply:
"False" is default so, I removed it :-)